politeiad/cache/cockroachdb/decred.go

```diff
@@ -653,6 +684,9 @@ func (d *decred) voteSummaries(tokens []string, bestBlock uint64) (map[string]de
 		}
 	}
 
+	// Update best block
+	d.bestBlockSet(bestBlock)
```

Comment:
What's the reasoning for updating the best block here?

Reply:
This was wrong, sorry. Removing this and now bb is updated properly, on plugin's load vote results command.